Alerting: Alert rule should wait For duration when execution error state is Alerting (#47052)
grafana/grafana: docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md

| Error or timeout option | Description                                                                                                                              |
| ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- |

> | Alerting                | Set alert rule state to `Alerting`. In Grafana 8.5 and later the alert rule must wait the duration for which the condition must be true. |

Reviewer: ```suggestion
| Alerting                | Set alert rule state to `Alerting`. From Grafana 8.5, the alert rule waits for the entire duration for which the condition is true before firing.|
```
